src/sql/src/func.rs

```diff
@@ -2003,6 +2003,25 @@ lazy_static! {
                     })
                 }), 939;
             },
+
+            "generate_subscripts" => Table {
+                params!(ArrayAny, Int32) => Operation::variadic(move |_ecx, exprs| {
+                    Ok(TableFuncPlan {
+                        func: TableFunc::GenerateSubscriptsArray,
+                        exprs,
+                        column_names: vec![Some("generate_subscripts".into())],
+                    })
+                }), 1192;
+                params!(ListAny, Int32) => Operation::variadic(move |_ecx, exprs| {
```

Comment:
So, lists don't really have dimensions: since lists are ragged, inner "dimensions" don't have a fixed size. The two functions that kinda pretend lists have dimensions (`list_ndims` and `list_length_max`) are marked as experimental for that reason. I think `generate_subscripts` on a list should take just one parameter, which is the list itself.